src/playercards/cards/FalseCovenant.ttslua

```diff
@@ -0,0 +1,14 @@
+buttonValue = "Cancel"
+buttonIcon = "token-curse"
+buttonColor = "#633A84E6"
+buttonFontSize = 300
+
+originParams = {
+  triggeringCard = "FalseCovenant",
+  redrawnTokenType = "random",
```

Comment:
Will this ever not be "random" except for Nkosi?

Reply:
Leaving it open for future expandability.